code/__HELPERS/global_lists.dm

```diff
@@ -66,9 +66,6 @@ GLOBAL_LIST_INIT(whitelisted_species, list(SPECIES_HUMAN))
 /// A list of ALL playable species, whitelisted, latejoin or otherwise.
 GLOBAL_LIST_EMPTY(playable_species)
 
-/// Poster designs (/datum/poster).
-GLOBAL_LIST_EMPTY(poster_designs)
```

Comment:
random posters should be done via `/obj/random` (see potted plants or other things for a example)